crates/next-api/src/app.rs

```diff
@@ -1113,6 +1116,7 @@ impl AppEndpoint {
                 let edge_function_definition = EdgeFunctionDefinition {
                     files: file_paths_from_root,
                     wasm: wasm_paths_to_bindings(wasm_paths_from_root),
+                    assets: paths_to_bindings(all_assets),
```

Comment:
Should this list all assets or only static assets referenced from the code?

Reply:
As far as I could find it doesn't matter if we list all of them or only the ones used by `new URL()` because this field is used during deployment of edge functions and it needs the full list of relevant files to run the route anyway.